install/v2plugin/Dockerfile

```diff
@@ -1,11 +1,13 @@
-# Docker v2plugin container with OVS / netplugin / netmaster 
+# Docker v2plugin container with OVS / netplugin / netmaster
 
 FROM alpine:3.5
-MAINTAINER Cisco Contiv (http://contiv.github.io/)
+MAINTAINER Cisco Contiv (https://contiv.github.io/)
 
-RUN mkdir -p /run/docker/plugins /etc/openvswitch /var/run/contiv/log \
-    && echo 'http://dl-cdn.alpinelinux.org/alpine/v3.4/main' >> /etc/apk/repositories \
-    && apk update && apk add openvswitch=2.5.0-r0 iptables
```

Comment:
is there a reason to remove apk update? Hope you did run make demo-v2plugin with this change.